docs/DeveloperGuide.md

```diff
@@ -14,25 +236,37 @@
 
 {Describe the target user profile}
 
+This product is for people who share kitchen space and ingredients with other cooks.
+
+
 ### Value proposition
 
 {Describe the value proposition: what problem does it solve?}
 
+Easy and intuitive way to keep track of ingredients you have in your kitchen. This helps avoid buying duplicated ingredients, reminds you of the ingredients you need, and gives a visualisation of the recipe timeline to ensure that advance preparation is done in time, eg marinating.
+
+
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...      | I want to ...                     | So that I can ...                            |
+|---------|---------------|-----------------------------------|----------------------------------------------|
+| v1.0    | beginner user | see all recipes                   | learn and try all recipes                    |
+| v1.0    | beginner user | add new recipes into the list     | learn and try out new recipes                |
+| v1.0    | beginner user | see all ingredient I have         |                                              |
+| v1.0    | beginner user | add ingredients to the empty list | add an item to my kitchen inventory          |
+| v2.0a   | amateur       | delete some recipes               | remove a recipe that I no longer want to use |
 
 ## Non-Functional Requirements
```

Comment:
Maybe you can include a section or subsection on error handling to make the user understand. Explain how errors or exceptional cases are handled in the application. This can include user input validation, error messages, and how developers can handle exceptions in their code.